src/libstd/sys_common/net.rs

```diff
@@ -53,6 +53,8 @@ use libc::MSG_NOSIGNAL;
               target_os = "haiku", target_os = "bitrig")))]
 const MSG_NOSIGNAL: c_int = 0x0;
 
+const MSG_PEEK: c_int = 0x2;
```

Comment:
This seems duplicated with the Windows definition? If this is the unix definition can it be pulled from libc?